### PR TITLE
style: update hp viz nav

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Update the hp-viz sub-navigation to maximize content space and make all the UI fully accessible on mobile view.

Screen Shot 2021-03-19 at 6 23 26 PM
Screen Shot 2021-03-19 at 6 24 14 PM
Screen Shot 2021-03-19 at 6 23 47 PM

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Review by hamidzr — COMMENTED** [review]
Looks good to me and works nicely. 

2 things that might be good chat about: 
- css vs js based responsive component
- potential use of https://ant.design/components/tabs/

File: webui/react/src/components/RadioGroup.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { useCallback, useState } from 'react';
```
[the remark this answers is not in the document]
Comment (hkang1) [reply webui/react/src/components/RadioGroup.stories.tsx:1]: Great feedback, made some updates to use a secondary set of tabs (card-style) to distinguish them more.

**Review by hamidzr — APPROVED** [review]
Great work 🚀  Two labeling comments:
- undo. maybe something like cancel would fit the behavior better. with undo I expect it to undo an applied action. 
- square and wrapped labels could seem flipped/confusing to the user

**Comment by hkang1** [comment]
> Great work 🚀 Two labeling comments:
> 
> * undo. I think cancel would fit the behavior better. with undo I expect it to undo an applied action.
> * square and wrapped labels could seem flipped/confusing to the user
> 
> 
> 

Great points, chatted offline briefly to land on the following:

- Reset (instead of Undo)
- Table View / Wrapped View instead of Squared View / Wrapped View

**Pull request merged**